etcd/etcdtest/mocketcd.go

```diff
@@ -38,6 +39,15 @@ func randPort() int {
 	return (1 << 15) + rand.Intn(1<<15) // #nosec
 }
 
+// Create a temporary directory for etcd's data-dir.
+func makeTempDir() (string, error) {
+	dir, err := os.MkdirTemp("", "etcdtest")
+	if err != nil {
+		return "", err
+	}
+	return dir, nil
+}
```

Comment:
This is for single use - lets just inline it.

Reply:
It is inline now with the latest commit.
Thanks for the feedback @AlexanderYastrebov